scripts/bnsd/deploy_proposals.js

```diff
@@ -0,0 +1,126 @@
+      throw new Error("Transaction failed", tx);
+    }
+  };
+}
+
+async function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+async function main() {
+  const connection = await connectionPromise;
+  const chainId = await connection.chainId();
+  const profile = new UserProfile();
+  const wallet = profile.addWallet(Ed25519HdWallet.fromMnemonic(adminMnemonic));
+  const identity = await profile.createIdentity(wallet.id, chainId, adminPath);
+  const signAndPost = createSignAndPoster(connection, profile);
+
+  const governorOptions = {
+    connection: connection,
+    identity: identity,
+  };
+  const governor = new Governor(governorOptions);
+
+  const proposalOptions = [
+    {
+      type: ProposalType.AddCommitteeMember,
+      title: "Add committee member",
+      description: "Add a committee member in more detail",
+      electionRuleId: electionRuleId,
+      committee: committeeId,
+      address: "tiov12shyht3pvvacvyee36w5844jkfh5s0mf4gszp9",
+      weight: 3,
+    },
+    {
+      type: ProposalType.AmendElectionRuleThreshold,
+      title: "Amend election rule threshold",
+      description: "Amend the election rule threshold in more detail",
+      electionRuleId: electionRuleId,
+      targetElectionRuleId: electionRuleId,
+      threshold: {
+        numerator: 3,
+        denominator: 4,
+      },
+    },
+    {
+      type: ProposalType.AmendElectionRuleQuorum,
+      title: "Amend election rule quorum",
+      description: "Amend the election rule quorum in more detail",
+      electionRuleId: electionRuleId,
+      targetElectionRuleId: electionRuleId,
+      quorum: {
+        numerator: 4,
+        denominator: 7,
+      },
+    },
+    {
+      type: ProposalType.AddValidator,
+      title: "Add validator",
+      description: "Add a validator in more detail",
+      electionRuleId: electionRuleId,
+      pubkey: identity.pubkey,
+      power: 2,
+    },
+    {
+      type: ProposalType.AmendProtocol,
+      title: "Amend protocol",
+      description: "Amend the protocol in more detail",
+      electionRuleId: electionRuleId,
+      text: "Give IOV devs master keys to all accounts",
+    },
+  ];
+
+  for (let i = 0; i < proposalOptions.length; ++i) {
+    const createProposalTx = await governor.buildCreateProposalTx({
+      ...proposalOptions[i],
+      startTime: new Date(Date.now() + 1000),
+    });
+    await signAndPost(createProposalTx);
+
+    await sleep(7000);
+
+    // Vote on 2/3 of the proposals
+    if (i % 3) {
+      const proposals = await governor.getProposals();
+      const voteTx = await governor.buildVoteTx(
+        proposals[proposals.length - 1].id,
+        // Vote Yes 1/2 of the time, No for the other 1/2
+        i % 2 ? VoteOption.Yes : VoteOption.No,
+      );
+      await signAndPost(voteTx);
+    }
+  }
+}
+
+main()
+  .catch(console.error)
+  .finally(async () => {
+    (await connectionPromise).disconnect();
+    process.exit(0);
+  });
```

Comment:
Since we don't have `ts-node` installed yet in this repo, I think it is good to have this pure .js